synapse/storage/databases/main/search.py

```diff
@@ -444,20 +454,25 @@ async def search_msgs(
         count_clauses = clauses
 
         if isinstance(self.database_engine, PostgresEngine):
+            search_query, tsquery_func = _parse_query_for_pgsql(
+                search_term, self.database_engine
+            )
             sql = (
-                "SELECT ts_rank_cd(vector, to_tsquery('english', ?)) AS rank,"
+                f"SELECT ts_rank_cd(vector, {tsquery_func}('english', ?)) AS rank,"
                 " room_id, event_id"
                 " FROM event_search"
-                " WHERE vector @@ to_tsquery('english', ?)"
+                f" WHERE vector @@  {tsquery_func}('english', ?)"
```

Comment:
Can we move these to multiline strings while we're here :innocent:

Reply:
I was planning to do a follow-up PR to update the entire module to multi-line strings. Would that be acceptable?